pypesto/storage/read_from_hdf5.py

```diff
@@ -0,0 +1,89 @@
+import h5py
+from ..result import Result
+from ..optimize.result import OptimizerResult
+from ..problem import Problem
+from ..objective import Objective
+
+
+def read_hdf5_optimization(f: h5py.File,
+                           start: h5py.Group) -> 'OptimizerResult':
```

Comment:
```suggestion
                           id: h5py.Group) -> 'OptimizerResult':
```
can the start be changed to "id" everywhere? Later, it might be non-consecutive numbers used here, so I think a generic "id" is better.

Reply:
id shadows some built-in function. Will use `optimization_id` instead